app/src/main/java/com/globant/example/mentorapp/home/presentation/view/adapter/ListUsersAdapter.java

```diff
@@ -50,16 +53,28 @@ public int getItemCount() {
         return users != null ? users.size() : context.getResources().getInteger(R.integer.zero_constant);
     }
 
-    public static class ViewHolder extends RecyclerView.ViewHolder {
+    static class ViewHolder extends RecyclerView.ViewHolder {
 
         private TextView userNameTextView;
         private ImageView userImage;
 
-        public ViewHolder(View itemView) {
+        public ViewHolder(View itemView, final Context context) {
```

Comment:
You do not need this context parameter, since you can get it from itemView, which is a View.